resources/views/layout.blade.php

```diff
@@ -0,0 +1,32 @@
+<Html>
+
+<head>
+    <title>
+        @yield('title')
+    </title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="row mt-5 justify-content-center">
+            <div class="col-md-8">
+                <div class="card">
+                    <div class="card-header">
+                        <h5>Create Memo</h5>
+                    </div>
+
+                    <div class="card-body">
+                        @yield('content')
+                    </div>
+                </div>
+                <div class=" mt-5  table-responsive">
+                    @yield('show_memos')
+                </div>
```

Comment:
it was better to have this in the memo.blade file & using this layout.blade file just for the site layout.
because it can be shared between each page.

its better for this file to have just reusable stuff, like:
<style> <script> or some structured <div> which would be used in any pages.

but for now, its ok. you can create an Issue an improve it whenever you want.